src/example/main.scss

```diff
@@ -89,12 +89,23 @@ article {
 }
 
 .example-with-click-me {
+  position: relative;
   text-align: center;
   &:hover {
     .click-me {
       animation: none;
     }
   }
+
+  .chart {
+    margin-right: 200px;
```

Comment:
Need these styles to get it looking right?
We'll need to figure out how to make these components integrate better without additional styling/CSS in the future.